cpp/dolfinx/fem/FiniteElement.cpp

```diff
@@ -143,19 +143,18 @@ FiniteElement::FiniteElement(const ufc_finite_element& ufc_element)
 
   if (is_basix_element(ufc_element))
   {
-    // FIXME: Find a better way than strings to initialise this Basix element
     if (ufc_element.needs_lattice_type)
     {
       _element = std::make_unique<basix::FiniteElement>(basix::create_element(
-          basix::element::str_to_type(_family.c_str()),
-          basix::cell::str_to_type(cell_shape.c_str()), ufc_element.degree,
+          (basix::element::family)ufc_element.basix_family,
```

Comment:
The 'proper' C++ way to do this would be `static_cast<basix::element::family>(ufc_element.basix_family)`.